ldes-fragmentisers/ldes-fragmentisers-substring/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentisers/substring/SubstringFragmentationStrategyWrapper.java

```diff
@@ -20,18 +22,21 @@ public class SubstringFragmentationStrategyWrapper implements FragmentationStrat
 	public FragmentationStrategy wrapFragmentationStrategy(ApplicationContext applicationContext,
```

Comment:
Why use the ApplicationContext here ?

Reply:
ApplicationContext is part of the general interface of `FragmentationStrategyWrapper`:
`FragmentationStrategy wrapFragmentationStrategy(ApplicationContext applicationContext,
			FragmentationStrategy fragmentationStrategy, ConfigProperties fragmentationProperties);`
Initial the idea was that each fragmentationstrategy would be able to get certain beans they need out from the ApplicationContext, e.g. for TimeBasedFragmentation it was needed to retrieve members to determine the max value of `generatedAtTime`, so TimeBasedFragmentation could get the MemberRepository bean.

Maybe to revise with the team as now all FragmentationStrategies use the same beans so maybe it's not very relevant anymore.